### DISCUSSION TITLE
Tropernic Intelligent Lock TY0A01

### DISCUSSION THREAD

**Discussion by Tomasz45** [Koenkk/zigbee2mqtt#19159] [Devices]
More and more people use smart, cheap Tuya ZigBee locks. We look forward to serving you at zigbee2mqtt.

**Comment by sibero80** [comment]
Are these locks compatible with Zigbee2Mqttt? https://es.aliexpress.com/item/1005005823725698.html?spm=a2g0s.imconversation.0.0.56bb3e5fnCL3Qo&algo_pvid=4e8d72ba-c6c9-4f9a-9ab0-bfcd49cc506f&algo_exp_id=4e8d72ba-c6c9-4f9a-9ab0-bfcd49cc506f-3&pdp_npi=4%40dis%21USD%21151.55%21121.24%21%21%211098.20%21%21%402103231116948074843371897e2758%2112000034487788740%21sea%21CO%21182506069%21S&curPageLogUid=HNMHO2jobfY5#nav-specification

**Comment by kkossev** [comment]
What is the shipping cost that you see for your country from this AE seller?

**Reply by sibero80** [reply]
Free
